smallrye-reactive-messaging-camel/src/test/java/io/smallrye/reactive/messaging/camel/outgoing/OutgoingCamelTest.java

```diff
@@ -132,9 +133,11 @@ void testWithBeanDeclaringASimpleProcessorThatForwardIncomingHeaders() throws Ex
 
             final CompletableFuture<Exchange> completableFuture = CompletableFuture
                     .supplyAsync(
```

Comment:
I would not use supplyAsync with a specific executor. The behavior is not the one you think when the level of parallelism is 1.